experiments/calibration/forward_model.jl

```diff
@@ -0,0 +1,503 @@
+ENV["CLIMACOMMS_CONTEXT"] = "SINGLETON" # force climacomms to use singleton context and not MPI
+
+import SciMLBase
+import TOML
+import ClimaComms
+ClimaComms.@import_required_backends
+import ClimaTimeSteppers as CTS
+using ClimaCore
+using ClimaUtilities.ClimaArtifacts
+import ClimaUtilities.OnlineLogging: WallTimeInfo, report_walltime
+import ClimaUtilities.TimeManager: ITime, date
+
+import ClimaDiagnostics
+import ClimaAnalysis
+import ClimaUtilities
+
+import ClimaUtilities.TimeVaryingInputs:
+    TimeVaryingInput, LinearInterpolation, PeriodicCalendar
+import ClimaUtilities.SpaceVaryingInputs: SpaceVaryingInput
+import ClimaUtilities.ClimaArtifacts: @clima_artifact
+import ClimaParams as CP
+
+using ClimaLand
+using ClimaLand.Snow
+using ClimaLand.Soil
+using ClimaLand.Canopy
+import ClimaLand
+import ClimaLand.Parameters as LP
+
+import ClimaCalibrate: forward_model, parameter_path, path_to_ensemble_member
+import ClimaCalibrate as CAL
+
+using Statistics
+using Dates
+import NCDatasets
+
+FT = Float64;
+time_interpolation_method = LinearInterpolation()
+context = ClimaComms.context()
+ClimaComms.init(context)
+device = ClimaComms.device()
+device_suffix = device isa ClimaComms.CPUSingleThreaded ? "cpu" : "gpu"
+
+function setup_prob(
+    t0,
+    tf,
+    Δt,
+    params,
+    model_start,
+    outdir;
+    nelements = (101, 15),
+)
+    earth_param_set = LP.LandParameters(FT)
+    # Set up parameters
+    p_names = collect(keys(params))
+    p_values = [params[name]["value"] for name in p_names]
+    params = (; zip(Symbol.(p_names), p_values)...)
+    (;
+        #        pc,
+        #        sc,
+        #        K_sat_plant,
+        #        a,
+        #        h_leaf,
+        #        α_snow,
+        #        α_soil_dry_scaler,
+        #        τ_leaf_scaler,
+        #        α_leaf_scaler,
+        #        α_soil_scaler,
```

Comment:
Why are these commented out?

Reply:
because we might want to calibrate with them (or not), maybe there is a more elegant solution than commenting out